src/components/embed_pages/HolidayEmbedPage.ts

```diff
@@ -0,0 +1,46 @@
+import { EmbedPage } from '../models/EmbedPage';
+import { client } from '../../index';
+
+export const data = new EmbedPage(
+    client,
+    false,
+    false,
+    false,
+    'holiday',
+    'Hoy no hay clase',
+    '',
+    [client.config.generalTextChannelID],
+    [
+        {
+            name: 'Frase insipradora del dia',
+            value: get_alan_kay_phrase(Math.floor(Math.random() * 15)),
```

Comment:
La funcion `get_alan_kay_phrase` lo unico que hace es devolver un elemento de una lista. No convendria volar la funcion, hacer que la lista quede como una variable global del modulo y dejar de hardcodear ese `15`?

```suggestion
            value: alan_kay_quotes[Math.floor(Math.random() * alan_kay_quotes.length)],
```
---

Otra alternativa quizas sea la de meter el numero aleatorio dentro de la funcion. Si quisieramos algo determinista simplemente accedemos a la lista al elemento correspondiente. Algo asi como:

```ts
alan_kay_quotes: string[] = [...];
const get_random_phrase = (phrases: string[]) => phrases[Math.floor(Math.random() * phrases.length)];
```

Y se la llamaria usando:
```suggestion
            value: get_random_phrase(alan_kay_quotes),
```

Reply:
No soy gran fan de las variables globales pero quizas es mi poca experiencia con js/ts jeje. Estoy de acuerdo en sacar ese 15 feo. Ahi lo hice de otra manera pero manteniendo la funcion, si crees que tu opcion es mejor de verdad que decime y lo cambiamos, confio mas en tu criterio

Reply:
Eso si que es tener fe jaja. Ahi vi el cambio. Para mi, esta mucho mejor ahora. Quizas como muy puntilloso le cambiaria el nombre a la funcion a algo asi como `get_alan_kay_phrase() -> get_random_alan_kay_phrase()`.

Con respecto a lo de que la lista sea "global", tiene que ver con que no me gusta que haya constantes tan grandes dentro de funciones. Ademas, pongo global entre comillas, porque si bien seria una variable global, seria global dentro del modulo. Si no se lo exporta, desde otro archivo no te enterarias que existe.
Por preferencia yo lo haria global, y ademas usaria `const` en lugar de `let`, pero son gustos y preferencias 😅.

Reply:
Dicho y hecho me gusto mucho mas cuando lo fui haciendo. Chequea que este todo en orden :)

Reply:
Dale, cierro esta conversacion y ahora reviso todo junto